### PR TITLE
fixes bangammers

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

SOMEONE DIDNT READ THE COMMENT BLOCK ABOUT FIXING THE SPELLING

## Why It's Good For The Game

god DAMN IT

## Changelog
:cl:
spellcheck: you can now stroke heads with a bangammer in peace
/:cl:

### CODE REVIEW

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -24,7 +24,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 */
 /obj/item/banhammer/attack(mob/M, mob/user)
 	if(user.zone_selected == BODY_ZONE_HEAD)
-		M.visible_message("<span class='danger'>[user] is stroking the head of [M] with a banhammer.</span>", "<span class='userdanger'>[user] is stroking your head with a banhammer.</span>", "<span class='hear'>You hear a banhammer stroking a head.</span>")
+		M.visible_message("<span class='danger'>[user] is stroking the head of [M] with a bangammer.</span>", "<span class='userdanger'>[user] is stroking your head with a bangammer.</span>", "<span class='hear'>You hear a bangammer stroking a head.</span>")
```
Comment (LoserWasTaken) [inline code/game/objects/items/weaponry.dm:27]: stupid

File: code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -24,7 +24,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 */
 /obj/item/banhammer/attack(mob/M, mob/user)
 	if(user.zone_selected == BODY_ZONE_HEAD)
-		M.visible_message("<span class='danger'>[user] is stroking the head of [M] with a banhammer.</span>", "<span class='userdanger'>[user] is stroking your head with a banhammer.</span>", "<span class='hear'>You hear a banhammer stroking a head.</span>")
+		M.visible_message("<span class='danger'>[user] is stroking the head of [M] with a bangammer.</span>", "<span class='userdanger'>[user] is stroking your head with a bangammer.</span>", "<span class='hear'>You hear a bangammer stroking a head.</span>")
 	else
 		M.visible_message("<span class='danger'>[M] has been banned FOR NO REISIN by [user]!</span>", "<span class='userdanger'>You have been banned FOR NO REISIN by [user]!</span>", "<span class='hear'>You hear a banhammer banning someone.</span>")
```
Comment (LoserWasTaken) [inline code/game/objects/items/weaponry.dm:29]: stupid

**Comment by SpaceManiac** [comment]
https://github.com/tgstation/tgstation/pull/46044/files#diff-12e81e9f891a99248867bd1d8a818e7b

File: code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (ATH1909) [inline code/game/objects/items/weaponry.dm:27]: according to the link posted by spacemaniac, the "is"es here were originally "are"s

File: code/game/objects/items/weaponry.dm [hunk 3]
```diff
@@ -24,7 +24,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 */
 /obj/item/banhammer/attack(mob/M, mob/user)
 	if(user.zone_selected == BODY_ZONE_HEAD)
-		M.visible_message("<span class='danger'>[user] is stroking the head of [M] with a banhammer.</span>", "<span class='userdanger'>[user] is stroking your head with a banhammer.</span>", "<span class='hear'>You hear a banhammer stroking a head.</span>")
+		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer.</span>", "<span class='userdanger'>[user] are stroking your head with a bangammer.</span>", "<span class='hear'>You hear a bangammer stroking a head.</span>")
 	else
 		M.visible_message("<span class='danger'>[M] has been banned FOR NO REISIN by [user]!</span>", "<span class='userdanger'>You have been banned FOR NO REISIN by [user]!</span>", "<span class='hear'>You hear a banhammer banning someone.</span>")
```
Comment (ATH1909) [inline code/game/objects/items/weaponry.dm:29]: should it be "banhammer" or "bangammer" in that last bit? I'm unfamiliar with the source material for this reference, so I don't know.

File: code/game/objects/items/weaponry.dm [hunk 4]
```diff
@@ -24,7 +24,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 */
 /obj/item/banhammer/attack(mob/M, mob/user)
 	if(user.zone_selected == BODY_ZONE_HEAD)
-		M.visible_message("<span class='danger'>[user] is stroking the head of [M] with a banhammer.</span>", "<span class='userdanger'>[user] is stroking your head with a banhammer.</span>", "<span class='hear'>You hear a banhammer stroking a head.</span>")
+		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer.</span>", "<span class='userdanger'>[user] are stroking your head with a bangammer.</span>", "<span class='hear'>You hear a bangammer stroking a head.</span>")
```
Comment (spookydonut) [inline code/game/objects/items/weaponry.dm:27]:
```suggestion
		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer.</span>", "<span class='userdanger'>[user] are stroking your head with a bangammer.</span>", "<span class='hear'>You hear a bangammer stroking a head.</span>") // see above comment
```

**Comment by Tharcoonvagh** [comment]
ban gamer

**Pull request merged**